ci/release/update-version.sh

```diff
@@ -37,8 +37,7 @@ sed_runner 's/'"  VERSION .*"'/'"  VERSION ${NEXT_FULL_TAG}"'/g' CMakeLists.txt
 
 # Python update
 sed_runner 's/'"rmm_version .*)"'/'"rmm_version ${NEXT_FULL_TAG})"'/g' python/CMakeLists.txt
-sed_runner "s/__version__ = .*/__version__ = \"${NEXT_FULL_TAG}\"/g" python/rmm/__init__.py
-sed_runner "s/^version = .*/version = \"${NEXT_FULL_TAG}\"/g" python/pyproject.toml
+sed_runner "/__version__ / s/= .*/= \"${NEXT_FULL_TAG}\"/g" python/rmm/_version.py
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏
```suggestion
sed_runner "/^__version__ / s/= .*/= \"${NEXT_FULL_TAG}\"/g" python/rmm/_version.py
```